hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/record/VectorClockTimestamp.java

```diff
@@ -129,7 +129,7 @@ static VectorClockTimestamp copyVector(VectorClockTimestamp vectorClockTimestamp
         return new VectorClockTimestamp(clocks);
     }
 
-    static boolean happenedBefore(VectorClockTimestamp x, VectorClockTimestamp y) {
+    public static boolean happenedBefore(VectorClockTimestamp x, VectorClockTimestamp y) {
```

Comment:
Can we make this method non-static? Then we can use it like clock1.happenedBefore(clock2)